src/main/java/com/ldtteam/structurize/util/BlueprintPositionInfo.java

```diff
@@ -0,0 +1,70 @@
+package com.ldtteam.structurize.util;
+
+import net.minecraft.nbt.CompoundNBT;
+import net.minecraft.util.math.BlockPos;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Contains all information we need for placement at a given position.
+ */
+public class BlueprintPositionInfo
+{
+    /**
+     * The position in the structure.
+     */
+    private final BlockPos pos;
```

Comment:
this is a relative schematic position right?

Reply:
yes